src/sentry/api/endpoints/project_release_details.py

```diff
@@ -31,14 +38,38 @@ def get(self, request, project, version):
         :pparam string version: the version identifier of the release.
         :auth: required
         """
+        with_health = request.GET.get("health") == "1"
+        summary_stats_period = request.GET.get("summaryStatsPeriod") or "48h"
+        health_stats_period = request.GET.get("healthStatsPeriod") or ("24h" if with_health else "")
```

Comment:
Maybe we could extract those constants somewhere? Now someone will have to remember to change them in 2 places and they can get out of sync.

Reply:
Not really something we do in the codebase from what I have seen.